go_api/domain/event.go

```diff
@@ -10,6 +10,7 @@ type Event struct {
 	Description string    `json:"description" gorm:"not null"`
 	CreatedAT   time.Time `json:"created_at" gorm:"not null"`
 	UpdatedAT   time.Time `json:"updated_at" gorm:"not null"`
+	Users       []User    `gorm:"many2many:event_user;"`
```

Comment:
```suggestion
	Users       []User    `json:"user,omitempty" gorm:"many2many:event_user;"`
```
にすると、nullはとりあえず消えそうですね。
https://qiita.com/nipe0324/items/55c5b51f37ce7f4679a1
参考記事添付しておくので、読んでみてください。